python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/experimental/test_columns_metadata.py

```diff
@@ -0,0 +1,141 @@
+            (TEST_SECOND_SCHEMA_NAME, TEST_TABLE_NAME),
+            (TEST_SECOND_SCHEMA_NAME, TEST_SECOND_TABLE_NAME),
+        ]:
+            table_spec = my_fivetran_assets.get_asset_spec(
+                AssetKey(
+                    [
+                        schema_name,
+                        table_name,
+                    ]
+                )
+            )
+            spec_table_schema = TableMetadataSet.extract(table_spec.metadata).column_schema
+
+            expected_spec_table_schema = TableSchema(
+                columns=[
+                    TableColumn(
+                        name="column_name_in_destination_1",
+                        type="",
+                        description=None,
+                        constraints=TableColumnConstraints(nullable=True, unique=False, other=[]),
+                        tags={},
+                    ),
+                    TableColumn(
+                        name="column_name_in_destination_2",
+                        type="",
+                        description=None,
+                        constraints=TableColumnConstraints(nullable=True, unique=False, other=[]),
+                        tags={},
+                    ),
+                ],
+                constraints=TableConstraints(other=[]),
+            )
+
+            assert spec_table_schema == expected_spec_table_schema
+
+        result = materialize(
+            [my_fivetran_assets],
+            resources={"fivetran": workspace},
+        )
+        assert result.success
+
+        for schema_name, table_name in [
+            (TEST_SCHEMA_NAME, TEST_TABLE_NAME),
+            (TEST_SCHEMA_NAME, TEST_SECOND_TABLE_NAME),
+            (TEST_SECOND_SCHEMA_NAME, TEST_TABLE_NAME),
+            (TEST_SECOND_SCHEMA_NAME, TEST_SECOND_TABLE_NAME),
+        ]:
+            table_schema_by_asset_key = {
+                event.materialization.asset_key: TableMetadataSet.extract(
+                    event.materialization.metadata
+                ).column_schema
+                for event in result.get_asset_materialization_events()
+                if event.materialization.asset_key
+                == AssetKey(
+                    [
+                        schema_name,
+                        table_name,
+                    ]
+                )
+            }
+            expected_table_schema_by_asset_key = {
+                AssetKey(
+                    [
+                        schema_name,
+                        table_name,
+                    ]
+                ): TableSchema(
+                    columns=[
+                        TableColumn("column_name_in_destination_1", type=""),
+                        TableColumn("column_name_in_destination_2", type=""),
+                    ]
+                ),
+            }
+
+            assert table_schema_by_asset_key == expected_table_schema_by_asset_key
+
+        captured = capsys.readouterr()
+        assert not re.search(
```

Comment:
weird assertion - there could be other errors that occur right

Reply:
Okay looking closer at the implementation I think this actually makes sense. Maybe just add a comment talking about the broadness of the assertion and that any errors caught should be logged like this

Reply:
I added a comment. The idea is to make sure that no exception occurred during the execution of `fetch_column_metadata` - if an exception occurs in `fetch_column_metadata`, a message is logged as a warning, but the exception is not raised. This is the current behavior in other fetch_column_metadata methods.